packages/scan/project.json

```diff
@@ -18,23 +19,18 @@
             "input": "",
             "output": ""
           }
-        ],
-        "buildableProjectDepsInPackageJsonType": "dependencies"
```

Comment:
Since `buildableProjectDepsInPackageJsonType` is gone, you probably have to use the dependency-checks ESLint rule to properly calculate the list of dependencies. Please check package.json in dist folder to make sure that we are not missing something.

Reply:
Thanks for raising very valid point! 
`@nx/dependency-checks` rule has configured, and projects' `package.json` files have been updated properly.
as a workaround  `ts-mockito` and `nock` dev deps are atm placed into `ignoredDependencies` rule exceptions list, but aim to address this issue in next prs with nx v19 update